qemu_v8.mk

```diff
@@ -285,6 +327,43 @@ $(ROOTFS_UGZ): u-boot buildroot | $(BINARIES_PATH)
 				-n "Root file system" \
 				-d $(ROOTFS_GZ) $(ROOTFS_UGZ)
 
+################################################################################
+# XEN
+################################################################################
+.PHONY: xen
+$(XEN_PATH)/xen/.config:
+	$(MAKE) -C $(XEN_PATH)/xen XEN_TARGET_ARCH=arm64 defconfig
+	cd $(XEN_PATH)/xen && \
+	tools/kconfig/merge_config.sh -m .config $(ROOT)/build/kconfigs/xen.conf
+
+xen: $(XEN_PATH)/xen/.config | $(BINARIES_PATH)
+	$(MAKE) -C $(XEN_PATH) dist-xen \
+	XEN_TARGET_ARCH=arm64 \
+	CONFIG_XEN_INSTALL_SUFFIX=.gz	\
+	CROSS_COMPILE="$(CCACHE)$(AARCH64_CROSS_COMPILE)"
+	ln -sf $(XEN_IMAGE) $(BINARIES_PATH)
```

Comment:
Is this symlink needed? It looks like `make XEN_BOOT=y run-only` works fine without `xen.efi` in `$(BINARIES_PATH)`.

Reply:
This was mainly required when I was using qemu options to load xen for edk2 boot. I will remove it for now and add it later when I add support to boot xen via edk2.

Reply:
Oh I see, one more configuration to test :open_mouth: :wink: 
Yes better to remove it if not used for now.